components/Dashboard/Modals/ProfileDialog/AccountDetails.js

```diff
@@ -96,10 +99,10 @@ const AccountDetails = ({ session, editMode, saveChanges, setSaveChanges, setEdi
         if (newImage) await handleUpdateImage();
         setSaveChanges(false);
         if (!passed) {
-          alert(message ?? 'You account info has been updated sucessfully 😃');
+          alert(message ?? t('SUCCESS_CHANGE_PROFILE'));
           setEditMode(true);
         } else {
-          alert(message ?? 'Internal Server Error');
+          alert(message ?? t('ERROR_SERVER'));
```

Comment:
can you prefix with `LABEL_` ?

Reply:
okk do u want all translations to be prefixed with LABEL_ ?

Reply:
Yes pls :3